.travis.yml

```diff
@@ -1,21 +1,24 @@
-sudo: false
-language: java
-jdk:
-  - openjdk6
-  - openjdk7
-  - oraclejdk7
+sudo: required
```

Comment:
This doesn't seem to be required for apt-get, is this for the benefit of the other sudo commands below?

Do you know if there is a downside to setting this?